### PR TITLE
Add Jupyterlab-graph-lsp v0.1.2

### PR DESCRIPTION
**Pull request by dfreeman06**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [ ] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [ ] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by dfreeman06** [comment]
@bollwyvl take a look when possible.

**Comment by bollwyvl** [comment]
@dfreeman06 yep, looks good, and i agree to help maintain... we'll see what CI says...

File: recipes/jupyterlab-graph-lsp/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+    # list
+    - jupyter labextension list
+    # validate
+    - jupyter labextension list 1>labextensions 2>&1
+    - grep -ie "@jupyrdf/jupyterlab-graph-lsp.*v{{ version.replace(".", "\\.") }}.*OK" labextensions
+    # test
+    - pytest --pyargs jupyterlab_graph_lsp --cov jupyterlab_graph_lsp --cov-report term-missing:skip-covered --no-cov-on-fail --cov-fail-under 58
+  requires:
+    - pip
+    - pytest-cov
```
Comment (bollwyvl) [inline recipes/jupyterlab-graph-lsp/meta.yaml:42]: needs:
```
- m2-grep  # [win]
```

**Comment by dfreeman06** [comment]
@conda-forge/staged-recipes @conda-forge/help-python ready for review! Thank you!

**Pull request merged**